sumologic/resource_sumologic_cse_entity_criticality_config_test.go

```diff
@@ -0,0 +1,99 @@
+package sumologic
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccSumologicSCEEntityCriticalityConfig_create(t *testing.T) {
+	var entityCriticalityConfig CSEEntityCriticalityConfig
+	nName := "New Entity Criticality"
+	nSeverityExpression := "severity + 2"
+	resourceName := "sumologic_cse_entity_criticality_config.entity_criticality_config"
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCSEEntityCriticalityConfigDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testCreateCSEEntityCriticalityConfigConfig(nName, nSeverityExpression),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckCSEEntityCriticalityConfigExists(resourceName, &entityCriticalityConfig),
+					testCheckEntityCriticalityConfigValues(&entityCriticalityConfig, nName, nSeverityExpression),
+					resource.TestCheckResourceAttrSet(resourceName, "id"),
+				),
+			},
+		},
+	})
+}
```

Comment:
we should add tests for importing and updating?

Reply:
added update test under
https://github.com/SumoLogic/terraform-provider-sumologic/pull/275/commits/eaf35c35121c9da0e0db4f53f0ffe2d8c937c064